packages/contracts/package.json

```diff
@@ -28,6 +28,7 @@
     "echidna": "echidna-test . --config echidna.config.yml",
     "prettier": "forge fmt",
     "deploy": "ts-node ./scripts/deploy/deploy.ts",
+    "deploy:diamond": "ts-node ./scripts/deploy/manager/DiamondInAll.ts",
```

Comment:
https://goerli.etherscan.io/tx/0x3a21916c1b2485bca894ee28dc6f00591cd070e6899fd95f0ebd34d701e37f8f
https://goerli.etherscan.io/tx/0x184a06d79dd58ea5fea7082e27763942af6aa9353060ccfe8e23ac039194e7e9

yarn workspace @ubiquity/contracts run deploy:diamond is failing at the last step. Please look at above 2 txs I've tried on your branch.

Reply to : https://github.com/ubiquity/ubiquity-dollar/pull/313#issuecomment-1341259363
> diamondCut function would handle all upgrading of the contract, no need to add upgrade script, imo

In terms of smart contract side, DiamondCutFacet::diamondCut will handle add/replace/remove actions properly but what we want here is high-level script. `deploy:diamond` will redeploy everything each time we  try to deploy which wouldn't be useful in real environment.

Reply:
> yarn workspace @ubiquity/contracts run deploy:diamond is failing at the last step.

Yeah, the contract was deployed, but failed when verifying. please use this command and .evn config
`yarn workspace @ubiquity/contracts ts-node ./scripts/deploy/manager/DiamondInAll.ts --network goerli`
`RPC_URL=https://goerli.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161`

Reply:
> `deploy:diamond` will redeploy everything each time we try to deploy which wouldn't be useful in real environment.



> In terms of smart contract side, DiamondCutFacet::diamondCut will handle add/replace/remove actions properly but what we want here is high-level script. `deploy:diamond` will redeploy everything each time we try to deploy which wouldn't be useful in real environment.

Hmmm, don't understand yet